pos_barcode_sync/README.rst

```diff
@@ -0,0 +1,53 @@
+================================
+ Barcode synchronization in POS
+================================
+
+Instantly updates partner data via longpoll connection in opened POSes if partner barcode was changed in the backend.
+
+Odoo updates partner data on opening the client list,
+this module updates partner data in options of already opened client list and if orders with changed partners are presence.
+It helps keep barcode data updated
```

Comment:
*in options* sounds weird.

> and if orders with changed partners are presence.

How this module affects existing orders? Once a customer is selected, it doesn't matter whether barcode of the customer is changed or not